coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -87,8 +82,16 @@ export function verifyPrescriptionBundle(bundle: fhir.Bundle): Array<errors.Vali
   const isRepeatDispensing = courseOfTherapyTypeCode === fhir.CourseOfTherapyTypeCode.CONTINUOUS_REPEAT_DISPENSING
   const repeatDispensingErrors = isRepeatDispensing ? verifyRepeatDispensingPrescription(medicationRequests) : []
 
+  const otherErrors = []
```

Comment:
feel like the name `incorrectValueErrors` was more descript

Reply:
didn't think it matched with a non-unique value error, so I thought this was better suited as a holder for single error types (instead of the other holders that check a bunch of stuff)

Reply:
maybe `invalidValueErrors`?

Reply:
but then I guess anything that throws an error is an invalid value

Reply:
yea lol thats true but still more accurate than `other`